frontend/model/contracts/group.js

```diff
@@ -504,8 +552,7 @@ DefineContract({
         }
       },
       process ({ data, meta }, { state, getters }) {
-        state.profiles[data.member].status = PROFILE_STATUS.REMOVED
-        state.profiles[data.member].departedDate = meta.createdDate
+        memberLeaves(state, data.member, meta.createdDate)
```

Comment:
I think maybe the following two lines got accidentally deleted when adding this line:
```
        state.profiles[data.member].status = PROFILE_STATUS.REMOVED
        state.profiles[data.member].departedDate = meta.createdDate
```
@taoeffect Can you confirm that these lines were not removed by mistake?

Reply:
@CrazedCoding haven't these two lines been moved into `memberLeaves`?

Reply:
@taoeffect This is correct, actually. Still getting used to reading the git diff's ><

Thanks for confirming.